ChangeLog

```diff
@@ -1,3 +1,19 @@
+Wed Sep 08 2020 Łukasz Plewa <lukasz.plewa@intel.com>
+
+	* Version 1.9.3-rc1
+	This release fixes a missing sfence in non-temporal
+	version of memcpy function
+	(https://github.com/pmem/pmdk/issues/5292),
+	and fixes a number of smaller bugs.
+
+	Detailed list of bug fixes:
+	common: fix missing sfence in non-temporal memcpy
```

Comment:
I suggest to stay with "-" at the beginning of each line (Look at previous rel).
Don't we want #5300 in this releases?